libs/tokens/src/updaters/TokensListsUpdater/helpers.ts

```diff
@@ -0,0 +1,26 @@
+import ms from 'ms.macro'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+
+import { ListState } from '../../types'
+
+const TOKENS_LISTS_UPDATER_INTERVAL = ms`6h`
+
+export const LAST_UPDATE_TIME_KEY = (chainId: SupportedChainId) => `tokens-lists-updater:last-update-time[${chainId}]`
+
+export const getIsTimeToUpdate = (chainId: SupportedChainId): boolean => {
+  const lastUpdateTime = +(localStorage.getItem(LAST_UPDATE_TIME_KEY(chainId)) || 0)
```

Comment:
Nitpick: would be worth to use an `atomWithStorage` instead of directly acessing `localStorage` for this?

Reply:
Also, probably should suffix it with a version (`:v0`)

Reply:
Makes sense, thanks!
Fixed: https://github.com/cowprotocol/cowswap/pull/3201/commits/7370a0bc351ec49d6026cde9c19da57692f0fe69